ankaios_sdk/_components/response.py

```diff
@@ -244,3 +258,44 @@ def __str__(self) -> str:
             str(instance_name) for instance_name in self.deleted_workloads]
         return f"Added workloads: {added_workloads}, " \
                f"Deleted workloads: {deleted_workloads}"
+
+
+class LogEntry:
+    """
+    Represents a log entry received from the Ankaios system.
+    """
+    def __init__(self, log: _ank_base.LogEntry) -> None:
+        """
+        Initializes the LogEntry with the given attributes.
+
+        Args:
+            log (_ank_base.LogEntry): The log entry from the Ankaios system.
+        """
+        self.workload_instance_name = WorkloadInstanceName(
+            log.workloadInstanceName.agentName,
+            log.workloadInstanceName.workloadName,
+            log.workloadInstanceName.workloadId
```

Comment:
You are using wrong names here.
```suggestion
            log.workloadName.agentName,
            log.workloadName.workloadName,
            log.workloadName.id
```